docker/start-network.sh

```diff
@@ -40,7 +40,7 @@ do
   # these will end up on drand1:8010, drand2:8020, drand3:8030, etc
   # note they map to the container's mapped ports, but the internal ports; internally the services still listen on 8080
   path=drand_docker_demo$i:80${i}0
-  docker run --rm --volume drand_docker_demo$i:/data/drand drandorg/go-drand:$docker_image_version generate-keypair --folder /data/drand/.drand --tls-disable --id default $path 1>/dev/null
+  docker run --rm --volume drand_docker_demo$i:/data/drand drandorg/go-drand:$docker_image_version generate-keypair --insecure --folder /data/drand/.drand --id default $path 1>/dev/null
```

Comment:
Do we really want the --insecure in the generate-keypair step here?

Reply:
yeah this is just a local demo; managing TLS certs in a docker-compose demo might be a pain